lambda/events/package.json

```diff
@@ -12,19 +12,19 @@
   "license": "ISC",
   "dependencies": {
     "@aws-sdk/client-lambda": "3.391.0",
-    "@sentry/serverless": "^6.18.2",
+    "@sentry/serverless": "7.64.0",
     "@types/node": "18.17.1",
     "aws-lambda": "^1.0.7",
-    "typescript": "5.1.6",
-    "node-fetch": "^2.6.7"
+    "node-fetch": "^2.6.7",
+    "typescript": "5.1.6"
```

Comment:
This isn't me but rather NPM reordering packages in alphabetical order, I assume.